blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueQueueContainer.java

```diff
@@ -1,7 +1,46 @@
 package io.jenkins.blueocean.rest.model;
 
+import org.kohsuke.stapler.WebMethod;
+import org.kohsuke.stapler.interceptor.JsonOutputFilter;
+import org.kohsuke.stapler.json.JsonResponse;
+import org.kohsuke.stapler.verb.POST;
+import org.kohsuke.stapler.verb.PUT;
+
 /**
  * @author Ivan Meredith
  */
 public abstract class BlueQueueContainer extends Container<BlueQueueItem> {
+    @WebMethod(name = "new")
+    @JsonResponse
+    @JsonOutputFilter(excludes = "state")
+    @POST
+    public abstract BlueQueueItemCreateResponse createItem();
```

Comment:
Following how we are naming get(String id) for getting an item, we should simply call it create(), Item is redundant.